.github/workflows/draft-release.yml

```
@@ -0,0 +1,149 @@
+      - name: Publish asc on GitHub Releases
+        uses: softprops/action-gh-release@a06a81a03ee405af7f2048a818ed3f03bbf83c7b # v2.5.0
+        with:
+          draft: true
+          fail_on_unmatched_files: true
+          token: ${{ secrets.CRYPTOBOT_RELEASE_TOKEN }}
+          files: |
+            cryptomator-*.tar.gz.asc
+
+  build-exe-and-msi:
+    needs: create-release
+    uses: ./.github/workflows/win-exe.yml
```

actionlint (1.7.12)

[error] input "revisionNum" is required by "./.github/workflows/win-exe.yml" reusable workflow

(workflow-call)